tasks/build-examples.js

```diff
@@ -16,6 +17,40 @@ var destDir = path.join(__dirname, '..', 'examples');
 var templatesDir = path.join(__dirname, '..', 'config', 'examples');
 
 /**
+ * Returns an array of classes that are explicitly required inside the source
+ * by calling 'goog.require(…)'
+ *
+ * @param {String} src The JavaScript sourcecode to search for goog.require.
+ * @returns {Array} An array of ol-classes that the source goog.requires.
+ */
+function getGoogRequires(src){
+  var googRequires = [];
+  var match = googRequiresRegEx.exec(src);
+  while (match) {
+    googRequires.push(match[1]);
+    match = googRequiresRegEx.exec(src);
+  }
+  return googRequires;
+}
+
+/**
+ * Takes an array of the names of required OpenLayers symbols and returns an
+ * HTML-snippet with an unordered list to the API-docs for the particular
+ * classes.
+ *
+ * @param {String[]} googRequires
```

Comment:
This isn't run through a doc linter (yet), but it would be good to stay consistent and use `Array.<string>` here.  Others would be `string` instead of `String` as well.